core/views.py

```diff
@@ -63,6 +63,32 @@ def post(self: "PostDetailView", request: HttpRequest, pk: int) -> HttpResponse:
         return HttpResponse(html_content)
 
 
+class AddPostView(LoginRequiredMixin, CreateView):
+    model = Post
+    form_class = PostForm
+    template_name = "post-add.html"
+    login_url = "login"
+
+    def get_form_kwargs(self: "AddPostView") -> dict[str, any]:
+        kwargs = super().get_form_kwargs()
+        # Set initial community to None
```

Comment:
Is this comment needed? Code is self explanatory.

Reply:
clean